parsers/jscript.c

```diff
@@ -2598,7 +2622,7 @@ static bool parseStatement (tokenInfo *const token, bool is_inside_class)
 					}
 
 					indexForName = isClassName (name) ?
-						makeClassTag (name, signature, NULL): 
```

Comment:
Remove trailing space

Reply:
This diff line starts from `-`.
This line removes the trailing space you introduced in 6d85089456 ( https://github.com/universal-ctags/ctags/commit/6d85089456ed215ce6b6a673744ae42ccc5e0e99#r78107639 ) .

Reply:
Doh!  Sorry!